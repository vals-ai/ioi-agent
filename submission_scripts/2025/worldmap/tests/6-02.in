version https://git-lfs.github.com/spec/v1
oid sha256:98b1691e71447f70d8a09405f7a9816dcb61f9932791be5ecdb986d3886e5d2a
size 45
