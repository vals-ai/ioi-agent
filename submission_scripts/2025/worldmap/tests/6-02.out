version https://git-lfs.github.com/spec/v1
oid sha256:7735bb92631d60152e51006830e48dc07ff01cc48fc4d1ffd8a1cc9403efbff9
size 79
